src/menu/rendersettingsmenustate.h

```diff
@@ -40,7 +42,8 @@ class RenderSettingsMenuState : public SelectedMenuState
   size_t m_currentListBox = 0;
```

Comment:
`m_currentListBox` is obsolete now, as it resembles the current tab index, please remove it.